src/libraries/Microsoft.PowerFx.Connectors/Execution/OpenApiJsonSerializer.cs

```diff
@@ -78,6 +82,11 @@ protected override void WriteDateValue(DateTime dateValue)
         protected override void WriteStringValue(string stringValue)
         {
             _writer.WriteStringValue(stringValue);
+        }
+
+        protected override async void WriteBlobValue(BlobValue blobValue)
+        {
+            _writer.WriteBase64StringValue(await blobValue.GetAsByteArrayAsync(_cancellationToken).ConfigureAwait(false));

```

Comment:
>_writer.WriteBase64StringValue(await blobValue.GetAsByteArrayAsync(_cancellationToken).ConfigureAwait(false)); [](http://example.com/codeflow?start=12&length=110)

Since our blob values store either bytes or directly the base64 encoded bytes, is there a way to see if it is the latter and avoid the decoding/re-encoding?